lib/matplotlib/backends/backend_svg.py

```diff
@@ -912,86 +922,46 @@ def _draw_text_as_path(self, gc, x, y, s, prop, angle, ismath, mtext=None):
         style = {}
         if color != '#000000':
             style['fill'] = color
-
         alpha = gc.get_alpha() if gc.get_forced_alpha() else gc.get_rgb()[3]
         if alpha != 1:
             style['opacity'] = short_float_fmt(alpha)
+        font_scale = fontsize / text2path.FONT_SCALE
+        attrib = {
+            'style': generate_css(style),
+            'transform': generate_transform([
+                ('translate', (x, y)),
+                ('rotate', (-angle,)),
+                ('scale', (font_scale, -font_scale))]),
+        }
+        writer.start('g', attrib=attrib)
 
         if not ismath:
             font = text2path._get_font(prop)
             _glyphs = text2path.get_glyphs_with_font(
                 font, s, glyph_map=glyph_map, return_new_glyphs_only=True)
             glyph_info, glyph_map_new, rects = _glyphs
+            self._update_glyph_map_defs(glyph_map_new)
```

Comment:
This does `char_id = self._adjust_char_id(char_id)`, which the original code did not. I assume that was an oversight in the original code?

Reply:
This was an oversight, because svg ids should not contain "%20" (https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/id), which is the only thing _adjust_char_id replaces.  Likely the oversight was benign, because I doubt any non-mathtext glyph names contained that substring, but it doesn't hurt to run them through _adjust_char_id, as well.